web/packages/teleport/src/Assist/Sidebar/Sidebar.tsx

```diff
@@ -46,34 +50,73 @@ const ChatHistoryTitle = styled.div`
   margin-bottom: 13px;
 `;
 
+const ChatHistoryItemButtons = styled.div`
+  position: absolute;
+  right: 5px;
+  top: 0;
+  bottom: 0;
+  display: none;
+  align-items: center;
+  justify-content: center;
+`;
+
+const ChatHistoryItemDeleteButton = styled.div`
```

Comment:
Could you make it a regular `<button>` instead of a div? Buttons come with a lot of accessibility stuff built in that divs just don't have or require implementing by hand. I think you should be able to use `ButtonIcon` here.

Reply:
@ryanclark Any opinions?